pkg/tools/golang/golang.go

```diff
@@ -47,6 +47,9 @@ type BuildConfig struct {
 
 	// Tags is go build tags.
 	Tags []string
+
+	// LdFlags are additional flags to pass to the go linker.
+	LdFlags []string
```

Comment:
Could you please do it more specific?
I would Imagine tat we have a structure here like this:
```
type BuildFlag struct {
    Name string
    Value string
}
```

Reply:
There's no structure at all, just list of string